color_pair_service.h

```diff
@@ -0,0 +1,9 @@
+
```

Comment:
The documentation (manual) can also be a 'service' we provide, yet it is not in color_pair_service.h.
Can we think of a more specific name to this file and its .c counterpart?

Reply:
Point noted. Please correct me if I am wrong. The function documentColorCode() can be declared in service.h since it is also a service provided. But the definition can be placed in the documenter.c. Am I right?